cookbook/import_export/create-connector.rst

```diff
@@ -88,6 +118,54 @@ You can run the job with the following command:
 
     php app/console akeneo:batch:job my_job_instance
 
-.. note::
+    [2016-07-07 16:48:35] batch.DEBUG: Job execution starting: startTime=, endTime=, updatedTime=, status=2, exitStatus=[UNKNOWN] , exitDescription=[], job=[my_job_instance] [] []
+    [2016-07-07 16:48:35] batch.INFO: Step execution starting: id=0, name=[dummy_step], status=[2], exitCode=[EXECUTING], exitDescription=[] [] []
+    [2016-07-07 16:48:35] batch.DEBUG: Step execution success: id= 1 [] []
+    [2016-07-07 16:48:35] batch.DEBUG: Step execution complete: id=1, name=[dummy_step], status=[1], exitCode=[EXECUTING], exitDescription=[] [] []
+    [2016-07-07 16:48:35] batch.DEBUG: Upgrading JobExecution status: startTime=2016-07-07T14:48:35+00:00, endTime=, updatedTime=, status=3, exitStatus=[UNKNOWN] , exitDescription=[], job=[my_job_instance] [] []
+    Export my_job_instance has been successfully executed.
+
+The ``--config`` option can be used to override the default values parameters, for instance, to change the file path,
+
+.. code-block:: bash
+
+    php app/console akeneo:batch:job my_job_instance --config='{"filePath":"\/tmp\/new_path.txt"}'
+
+Configure the UI for our JobParameters
+--------------------------------------
+
+For now, the job is usable in command line but it can't be configured through the UI.
+
+To do so, we need to write a service providing the form type configuration for each parameter of our JobParameters.
+
+In our case we want to display a text field to fulfill the file path, please note that the validation contraint defined in the previous service will also be used here.
```

Comment:
a text field where to fill in the path. The validation constraint defined earlier will also apply here.